src/System.Net.Http/tests/FunctionalTests/IdnaProtocolTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Net.Test.Common;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public abstract class IdnaProtocolTests : HttpClientTestBase
+    {
+        private const string s_IdnaHost = "âbçdë.com";
```

Comment:
We try not to add Unicode characters to source files.  Instead, we use "\uXXXX" syntax in the string literals.

Reply:
Is Unicode allowed in comments?  I'm guessing not.